### PR TITLE
Add test, app layer and README for FastQC. Add test output data. 

### LINKED ISSUE #546: [Docker Workshop]: fastqc
### What container needs an update?

I know all of these containers are awesome, but...

### PR DESCRIPTION
**Pull request by poojasgupta**
Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
-  [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

This pull request Fixes #546

### CODE REVIEW

**Comment by kapsakcj** [comment]
Hi Pooja,

Could you please alter lines 19 and 20 to remove the `1` after `maintainer`? This should resolve the error in the GitHub Actions testing workflow. Like this:
```Dockerfile
LABEL maintainer="Abigail Shockey"
LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
```
If you'd like you can add yourself as maintainer3, but this is optional.

Very nice usage of multi-stage builds on line 45 and below!

And lastly it looks like all the tests passed as expected 🎉 

Very nice work. After you make that change, we can get this merged and re-deploy the docker image to dockerhub and quay

**Comment by poojasgupta** [comment]
Hi Curtis, 

Thanks for the quick feedback. 
I have made the change as requested so hopefully it should not fail again.
Is this a new format to not have 'LABEL maintainer1' and just have 'LABEL maintainer'? I am curious because I did not actually edit this section and copied as is from the previous dockerfile. 

I am glad that all other tests passed successfully.

**Comment by kapsakcj** [comment]
No problem. Yes, it's a newer format requirement that we implemented within the last few months.

**Review by kapsakcj — APPROVED** [review]

**Pull request merged**
